src/styles/index.scss

```diff
@@ -6,7 +6,17 @@ html,
 body {
   color: #fff;
   background-color: #000;
-  height: 100%;
+  // height: 100%;
+}
+
+[data-reach-dialog-overlay] {
+  color: black;
+  background: white;
+  width: 100%;
+}
+[data-reach-dialog-content] {
+  width: 90% !important;
+  // height: 90%;
```

Comment:
is it something useful for later ?

Reply:
Nope, forgot to delete it, was for a test